arangod/MMFiles/MMFilesCompactorThread.cpp

```diff
@@ -486,10 +485,15 @@ void MMFilesCompactorThread::compactDatafiles(LogicalCollection* collection,
   }
 
   // now compact all datafiles
+  physical->_datafileStatistics.compactionRun();
+  uint64_t noCombined = 0;
   for (size_t i = 0; i < n; ++i) {
     auto compaction = toCompact[i];
     MMFilesDatafile* df = compaction._datafile;
 
+    MMFilesDatafileStatisticsContainer dfi = static_cast<MMFilesCollection*>(collection->getPhysical())->_datafileStatistics.get(df->fid());
+    physical->_datafileStatistics.compactionRead(dfi.sizeAlive +  dfi.sizeDead);
```

Comment:
Not accurate... Should rather be the size of the datafile and not sizeAlive + sizeDead. Note that there is also padding, plus the removal operations.